app/views/managers/show.html.erb

```diff
@@ -0,0 +1,19 @@
+<div class="container">
+  <% @users.each do |user| %>
+    <div class="row d-flex mt-3">
+      <div class="col-4">
+        <%= user.name %>
+      </div>
+
+      <div class="col-4">
+        <%= user.submissions.count %>
+      </div>
+
+      <div class="col-4">
+        <button class="btn btn-primary">
+        <%= link_to "Check Out", show_request_manager_path(user_id: user.id), class: "btn" %>
+        </button>
+      </div>
+    </div>
```

Comment:
why the manager show has all other users naming convention and action does not match